src/org.hdfgroup.object/hdf/object/h5/H5Datatype.java

```diff
@@ -2275,6 +2303,198 @@ public long getReferenceType() throws HDF5Exception {
         return -1;
     }
 
+    /**
+     * Describes the dataset object description for a 1.10 reference.
+     *
+     * @param container
+     *                  the dataset/attribute with the reference
+     * @param refarr
+     *                  the reference datatype data to be checked.
+     *
+     * @return the dataset reference object description.
+     */
+    public static String descReferenceObject(long container, byte[] refarr) {
```

Comment:
Seems like these routines should probably be part of the H5ReferenceType class and not H5Datatype.

Reply:
If we didn't have to support 1.10 based HDFView - yes. OR I add ReferenceType class to 1.10 based object lib.